### ISSUE TITLE
(Debugger) ECHO is limited to 55 chars

### ISSUE THREAD

**Issue by TommyGH** [AppleWin/AppleWin#481]
Per Mike's request in #476, I'm creating another Issue.
Title says it all.  Can only display 55 characters with ECHO command.

**Comment by Michaelangel007** [comment]
Repro.:

```
echo " 23456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789ABC"
```

**Comment by Michaelangel007** [comment]
* `CMD_OUTPUT_ECHO`
* `CmdOutputEcho()`

**Comment by Michaelangel007** [comment]
Bug not checking if iTokenSrc is `TOKEN_QUOTE_DOUBLE`

			if (nBuf > 0)
			{
				nLen = MIN( nBuf, MAX_ARG_LEN-1 );

**Comment by Michaelangel007** [comment]
Do we need to cap against `CONSOLE_WIDTH` ?

**Comment by Michaelangel007** [comment]
NOTE: Need to merge #480 before we can bump debugger version & update docs/Debugger_Changelog.txt

**Comment by Michaelangel007** [comment]
**sigh** Need to spin off a new bug ... console input dropping chars

* run output clipped to 61 chars
```
 23456789_123456789_123456789_123456789_123456789_123456789_1
```

* echo output clipped to 59 chars

```
 23456789_123456789_123456789_123456789_123456789_123456789_123
```
